app/workers/zomato_data_grabbing_worker.rb

```diff
@@ -0,0 +1,73 @@
+require 'json'
+class ZomatoDataGrabbingWorker
```

Comment:
Too many logic here, I would extract all the connection logic into a separate class that handles fetching and parsing responses. That's way it also will be easier better to test.

Reply:
Where you will usually put such file in the ROR infa? as a Module\Helper\Other?

Reply:
worker dir is fine